### PR TITLE
chore: update UI for CreateOrJoinScreen component

### LINKED ISSUE #705: (Front) Project Management Screen
**Description**
If user Project Coordinator or Participant attempts to leave project, display Project Management Screen

**Screen is:**
Mapeo Logo
(Create Project)
(Practice Mode)
Tap here to Joint a project (redirects to the Joint Project flow)

**Design**
[Here
](https://www.figma.com/file/ZOsM89L0xy0NUOCkuksJgq/OTF-%2B-UXFund?node-id=46889%3A1828)

**Points**
5pts

### PR DESCRIPTION
**Pull request by achou11**
Closes #705 

Notes:
- the icon is not exactly the same as what the mockup uses because I couldn't find it in the MD directory (maybe it's not an MD icon? idk)
- pressing the practice mode button just displays an alert about the feature not being implemented yet
---

Preview:

Screenshot_20210826-153847

### CODE REVIEW

File: src/frontend/screens/Onboarding/CreateOrJoinScreen.tsx [hunk 1]
```diff
@@ -4,101 +4,135 @@
  *   - Set `FEATURE_ONBOARDING=true` when running/building
  *   - Manually change the context value in `SettingsContext.tsx`
  */
-import React from "react";
-import { View, Text, StyleSheet, Image, Alert } from "react-native";
+import * as React from "react";
```
Comment (ErikSin) [inline src/frontend/screens/Onboarding/CreateOrJoinScreen.tsx:7]: I suppose we should start doing this for all ts files as it is best practice

File: src/frontend/screens/Onboarding/CreateOrJoinScreen.tsx [hunk 2]
```diff
@@ -4,101 +4,135 @@
...
 
-const navOptions: NavigationStackOptions = {
-  headerShown: false,
+const showWipAlert = () => {
+  Alert.alert("Work in progress", "This feature has not been implemented yet", [
+    {
+      text: "Ok",
+      onPress: () => {},
+    },
+  ]);
```
Comment (ErikSin) [inline src/frontend/screens/Onboarding/CreateOrJoinScreen.tsx]: Lets just make this a reuasable function because we are using it in alot of places

**Review by ErikSin — APPROVED** [review]
I am going to approve this, but if you could make the WIP notification a reusable function before we merge it that would be great :)

File: src/frontend/screens/Onboarding/CreateOrJoinScreen.tsx (hunk 1 above)
Comment (achou11) [reply src/frontend/screens/Onboarding/CreateOrJoinScreen.tsx:7]: yeah I've just been implementing the change as I come across it

**Pull request merged**